### ISSUE TITLE
install Applikation 'lxd' ERROR

### ISSUE THREAD

**Issue by DVSB** [YunoHost/lxd_img_builder#2]
args:
  app: lxd
  force: false
  label: LXD
  no_remove_on_failure: false
ended_at: 2022-05-30 12:30:00.706511
env:
  YNH_APP_BASEDIR: /var/cache/yunohost/app_tmp_work_dirs/app_w30y5dx0
  YNH_APP_ID: lxd
  YNH_APP_INSTANCE_NAME: lxd
  YNH_APP_INSTANCE_NUMBER: '1'
  YNH_APP_MANIFEST_VERSION: 5.1~ynh1
  YNH_ARCH: amd64
error: Im Installationsscript ist ein Fehler aufgetreten
interface: api
operation: app_install
parent: null
related_to:
- - app
  - lxd
started_at: 2022-05-30 12:29:49.808106
success: false
yunohost_version: 4.3.6.3

============

2022-05-30 14:29:49,818: INFO - lxd wird installiert...
2022-05-30 14:29:49,884: DEBUG - Nothing to update in LDAP
2022-05-30 14:29:49,884: DEBUG - Berechtigung 'lxd.main' aktualisiert
2022-05-30 14:29:49,916: DEBUG - Vollständiges Log dieser Operation: '<a href="#/tools/logs/20220530-122949-permission_url-lxd" style="text-decoration:underline">Aktualisiere URL, die mit der Berechtigung 'lxd' verknüpft ist</a>'
2022-05-30 14:29:49,993: DEBUG - The permission database has been resynchronized
2022-05-30 14:29:50,040: DEBUG - Konfiguration von SSOwat neu erstellt
2022-05-30 14:29:50,087: DEBUG - Berechtigung 'lxd.main' erstellt
2022-05-30 14:29:50,088: DEBUG - Vollständiges Log dieser Operation: '<a href="#/tools/logs/20220530-122949-permission_create-lxd" style="text-decoration:underline">Erstelle Berechtigung 'lxd'</a>'
2022-05-30 14:29:50,110: DEBUG - Executing command '['sh', '-c', '/bin/bash -x "./install"  7>&1']'
2022-05-30 14:29:50,123: DEBUG - + source _common.sh
2022-05-30 14:29:50,123: DEBUG - ++ pkg_dependencies='acl attr autoconf dnsmasq-base git libacl1-dev libcap-dev liblxc1 lxc-dev libsqlite3-dev libtool libudev-dev liblz4-dev libuv1-dev make pkg-config rsync squashfs-tools tar tcl xz-utils ebtables libapparmor-dev libseccomp-dev libcap-dev lvm2 thin-provisioning-tools btrfs-progs busybox|busybox-static libattr1-dev libuv1 libdevmapper-event1.02.1 dmeventd lxc tcl8.6'
2022-05-30 14:29:50,124: DEBUG - + source /usr/share/yunohost/helpers
2022-05-30 14:29:50,124: DEBUG - +++ set +o
2022-05-30 14:29:50,125: DEBUG - +++ grep xtrace
2022-05-30 14:29:50,127: DEBUG - ++ readonly 'XTRACE_ENABLE=set -o xtrace'
2022-05-30 14:29:50,128: DEBUG - ++ XTRACE_ENABLE='set -o xtrace'
2022-05-30 14:29:50,180: DEBUG - + ynh_abort_if_errors
2022-05-30 14:29:50,181: DEBUG - + trap ynh_exit_properly EXIT
2022-05-30 14:29:50,181: DEBUG - + app=lxd
2022-05-30 14:29:50,282: INFO - [....................] > Storing installation settings...
2022-05-30 14:29:50,410: INFO - [++++++..............] > Installing dependencies...
2022-05-30 14:29:50,411: DEBUG - + ynh_exec_warn_less ynh_install_app_dependencies acl attr autoconf dnsmasq-base git libacl1-dev libcap-dev liblxc1 lxc-dev libsqlite3-dev libtool libudev-dev liblz4-dev libuv1-dev make pkg-config rsync squashfs-tools tar tcl xz-utils ebtables libapparmor-dev libseccomp-dev libcap-dev lvm2 thin-provisioning-tools btrfs-progs 'busybox|busybox-static' libattr1-dev libuv1 libdevmapper-event1.02.1 dmeventd lxc tcl8.6
2022-05-30 14:29:50,412: DEBUG - + [[ 36 -eq 1 ]]
2022-05-30 14:29:50,412: DEBUG - + ynh_install_app_dependencies acl attr autoconf dnsmasq-base git libacl1-dev libcap-dev liblxc1 lxc-dev libsqlite3-dev libtool libudev-dev liblz4-dev libuv1-dev make pkg-config rsync squashfs-tools tar tcl xz-utils ebtables libapparmor-dev libseccomp-dev libcap-dev lvm2 thin-provisioning-tools btrfs-progs 'busybox|busybox-static' libattr1-dev libuv1 libdevmapper-event1.02.1 dmeventd lxc tcl8.6
2022-05-30 14:29:50,412: DEBUG - + local 'dependencies=acl attr autoconf dnsmasq-base git libacl1-dev libcap-dev liblxc1 lxc-dev libsqlite3-dev libtool libudev-dev liblz4-dev libuv1-dev make pkg-config rsync squashfs-tools tar tcl xz-utils ebtables libapparmor-dev libseccomp-dev libcap-dev lvm2 thin-provisioning-tools btrfs-progs busybox|busybox-static libattr1-dev libuv1 libdevmapper-event1.02.1 dmeventd lxc tcl8.6'
2022-05-30 14:29:50,413: DEBUG - ++ echo 'acl attr autoconf dnsmasq-base git libacl1-dev libcap-dev liblxc1 lxc-dev libsqlite3-dev libtool libudev-dev liblz4-dev libuv1-dev make pkg-config rsync squashfs-tools tar tcl xz-utils ebtables libapparmor-dev libseccomp-dev libcap-dev lvm2 thin-provisioning-tools btrfs-progs busybox|busybox-static libattr1-dev libuv1 libdevmapper-event1.02.1 dmeventd lxc tcl8.6'
2022-05-30 14:29:50,413: DEBUG - ++ sed 's/\([^\<=\>]\)\ \([^(]\)/\1, \2/g'
2022-05-30 14:29:50,415: DEBUG - + dependencies='acl, attr, autoconf, dnsmasq-base, git, libacl1-dev, libcap-dev, liblxc1, lxc-dev, libsqlite3-dev, libtool, libudev-dev, liblz4-dev, libuv1-dev, make, pkg-config, rsync, squashfs-tools, tar, tcl, xz-utils, ebtables, libapparmor-dev, libseccomp-dev, libcap-dev, lvm2, thin-provisioning-tools, btrfs-progs, busybox|busybox-static, libattr1-dev, libuv1, libdevmapper-event1.02.1, dmeventd, lxc, tcl8.6'
2022-05-30 14:29:50,416: DEBUG - + local 'dependencies=acl, attr, autoconf, dnsmasq-base, git, libacl1-dev, libcap-dev, liblxc1, lxc-dev, libsqlite3-dev, libtool, libudev-dev, liblz4-dev, libuv1-dev, make, pkg-config, rsync, squashfs-tools, tar, tcl, xz-utils, ebtables, libapparmor-dev, libseccomp-dev, libcap-dev, lvm2, thin-provisioning-tools, btrfs-progs, busybox | busybox-static, libattr1-dev, libuv1, libdevmapper-event1.02.1, dmeventd, lxc, tcl8.6'
2022-05-30 14:29:50,416: DEBUG - + local manifest_path=/var/cache/yunohost/app_tmp_work_dirs/app_w30y5dx0/manifest.json
2022-05-30 14:29:50,417: DEBUG - ++ jq -r .version /var/cache/yunohost/app_tmp_work_dirs/app_w30y5dx0/manifest.json
2022-05-30 14:29:50,423: DEBUG - + local version=5.1~ynh1
2022-05-30 14:29:50,423: DEBUG - + '[' -z 5.1~ynh1 ']'
2022-05-30 14:29:50,423: DEBUG - + '[' 5.1~ynh1 == null ']'
2022-05-30 14:29:50,424: DEBUG - + local dep_app=lxd
2022-05-30 14:29:50,424: DEBUG - + [[ acl, attr, autoconf, dnsmasq-base, git, libacl1-dev, libcap-dev, liblxc1, lxc-dev, libsqlite3-dev, libtool, libudev-dev, liblz4-dev, libuv1-dev, make, pkg-config, rsync, squashfs-tools, tar, tcl, xz-utils, ebtables, libapparmor-dev, libseccomp-dev, libcap-dev, lvm2, thin-provisioning-tools, btrfs-progs, busybox | busybox-static, libattr1-dev, libuv1, libdevmapper-event1.02.1, dmeventd, lxc, tcl8.6 =~ [<=>] ]]
2022-05-30 14:29:50,425: DEBUG - ++ echo acl, attr, autoconf, dnsmasq-base, git, libacl1-dev, libcap-dev, liblxc1, lxc-dev, libsqlite3-dev, libtool, libudev-dev, liblz4-dev, libuv1-dev, make, pkg-config, rsync, squashfs-tools, tar, tcl, xz-utils, ebtables, libapparmor-dev, libseccomp-dev, libcap-dev, lvm2, thin-provisioning-tools, btrfs-progs, busybox '|' busybox-static, libattr1-dev, libuv1, libdevmapper-event1.02.1, dmeventd, lxc, tcl8.6
2022-05-30 14:29:50,425: DEBUG - ++ grep -oP '(?<=php)[0-9.]+(?=-|\>)'
2022-05-30 14:29:50,426: DEBUG - ++ sort -u
2022-05-30 14:29:50,429: DEBUG - + local specific_php_version=
2022-05-30 14:29:50,429: DEBUG - + [[ '' != \7\.\3 ]]
2022-05-30 14:29:50,429: DEBUG - + [[ true == \t\r\u\e ]]
2022-05-30 14:29:50,430: DEBUG - + YNH_INSTALL_APP_DEPENDENCIES_REPLACE=false
2022-05-30 14:29:50,430: DEBUG - + grep --quiet php
2022-05-30 14:29:50,432: DEBUG - + cat
2022-05-30 14:29:50,435: DEBUG - + ynh_package_install_from_equivs /tmp/lxd-ynh-deps.control
2022-05-30 14:29:50,435: DEBUG - + local controlfile=/tmp/lxd-ynh-deps.control
2022-05-30 14:29:50,436: DEBUG - ++ grep '^Package: ' /tmp/lxd-ynh-deps.control
2022-05-30 14:29:50,437: DEBUG - ++ cut '--delimiter= ' --fields=2
2022-05-30 14:29:50,439: DEBUG - + local pkgname=lxd-ynh-deps
2022-05-30 14:29:50,441: DEBUG - ++ grep '^Version: ' /tmp/lxd-ynh-deps.control
2022-05-30 14:29:50,441: DEBUG - ++ cut '--delimiter= ' --fields=2
2022-05-30 14:29:50,444: DEBUG - + local pkgversion=5.1~ynh1
2022-05-30 14:29:50,445: DEBUG - + [[ -z lxd-ynh-deps ]]
2022-05-30 14:29:50,445: DEBUG - + [[ -z 5.1~ynh1 ]]
2022-05-30 14:29:50,445: DEBUG - + ynh_package_update
2022-05-30 14:29:50,446: DEBUG - + ynh_apt update
2022-05-30 14:29:50,446: DEBUG - + ynh_wait_dpkg_free
2022-05-30 14:29:50,747: DEBUG - + return 0
2022-05-30 14:29:50,748: DEBUG - + LC_ALL=C
2022-05-30 14:29:50,748: DEBUG - + DEBIAN_FRONTEND=noninteractive
2022-05-30 14:29:50,748: DEBUG - + apt-get --assume-yes --quiet -o=Acquire::Retries=3 -o=Dpkg::Use-Pty=0 update
2022-05-30 14:29:51,048: DEBUG - Hit:1 http://deb.debian.org/debian buster InRelease
2022-05-30 14:29:51,049: DEBUG - Hit:2 http://security.debian.org/debian-security buster/updates InRelease
2022-05-30 14:29:51,069: DEBUG - Hit:3 http://deb.debian.org/debian buster-updates InRelease
2022-05-30 14:29:51,089: DEBUG - Hit:4 http://forge.yunohost.org/debian buster InRelease
2022-05-30 14:29:51,149: DEBUG - Hit:5 https://packages.sury.org/php buster InRelease
2022-05-30 14:29:53,325: DEBUG - Reading package lists...
2022-05-30 14:29:53,368: DEBUG - ++ mktemp --directory
2022-05-30 14:29:53,370: DEBUG - + local TMPDIR=/tmp/tmp.emC1brpNLq
2022-05-30 14:29:53,370: DEBUG - + echo 10
2022-05-30 14:29:53,371: DEBUG - + ynh_wait_dpkg_free
2022-05-30 14:29:53,761: DEBUG - + return 0
2022-05-30 14:29:53,761: DEBUG - + cp /tmp/lxd-ynh-deps.control /tmp/tmp.emC1brpNLq/control
2022-05-30 14:29:53,763: DEBUG - + cd /tmp/tmp.emC1brpNLq
2022-05-30 14:29:53,764: DEBUG - + LC_ALL=C
2022-05-30 14:29:53,764: DEBUG - + equivs-build ./control
2022-05-30 14:29:55,291: DEBUG - + tee ./dpkg_log
2022-05-30 14:29:55,292: DEBUG - + LC_ALL=C
2022-05-30 14:29:55,292: DEBUG - + dpkg --force-depends --install ./lxd-ynh-deps_5.1~ynh1_all.deb
2022-05-30 14:29:55,483: DEBUG - Selecting previously unselected package lxd-ynh-deps.
2022-05-30 14:29:55,540: DEBUG - (Reading database ... 53037 files and directories currently installed.)
2022-05-30 14:29:55,543: DEBUG - Preparing to unpack .../lxd-ynh-deps_5.1~ynh1_all.deb ...
2022-05-30 14:29:55,630: DEBUG - Unpacking lxd-ynh-deps (5.1~ynh1) ...
2022-05-30 14:29:56,176: DEBUG - dpkg: lxd-ynh-deps: dependency problems, but configuring anyway as you requested:
2022-05-30 14:29:56,177: DEBUG -  lxd-ynh-deps depends on attr; however:
2022-05-30 14:29:56,177: DEBUG -   Package attr is not installed.
2022-05-30 14:29:56,177: DEBUG -  lxd-ynh-deps depends on libacl1-dev; however:
2022-05-30 14:29:56,178: DEBUG -   Package libacl1-dev is not installed.
2022-05-30 14:29:56,178: DEBUG -  lxd-ynh-deps depends on libcap-dev; however:
2022-05-30 14:29:56,178: DEBUG -   Package libcap-dev is not installed.
2022-05-30 14:29:56,179: DEBUG -  lxd-ynh-deps depends on liblxc1; however:
2022-05-30 14:29:56,179: DEBUG -   Package liblxc1 is not installed.
2022-05-30 14:29:56,179: DEBUG -  lxd-ynh-deps depends on lxc-dev; however:
2022-05-30 14:29:56,180: DEBUG -   Package lxc-dev is not installed.
2022-05-30 14:29:56,180: DEBUG -  lxd-ynh-deps depends on libsqlite3-dev; however:
2022-05-30 14:29:56,180: DEBUG -   Package libsqlite3-dev is not installed.
2022-05-30 14:29:56,180: DEBUG -  lxd-ynh-deps depends on libudev-dev; however:
2022-05-30 14:29:56,181: DEBUG -   Package libudev-dev is not installed.
2022-05-30 14:29:56,181: DEBUG -  lxd-ynh-deps depends on liblz4-dev; however:
2022-05-30 14:29:56,181: DEBUG -   Package liblz4-dev is not installed.
2022-05-30 14:29:56,182: DEBUG -  lxd-ynh-deps depends on libuv1-dev; however:
2022-05-30 14:29:56,182: DEBUG -   Package libuv1-dev is not installed.
2022-05-30 14:29:56,182: DEBUG -  lxd-ynh-deps depends on pkg-config; however:
2022-05-30 14:29:56,182: DEBUG -   Package pkg-config is not installed.
2022-05-30 14:29:56,183: DEBUG -  lxd-ynh-deps depends on squashfs-tools; however:
2022-05-30 14:29:56,183: DEBUG -   Package squashfs-tools is not installed.
2022-05-30 14:29:56,183: DEBUG -  lxd-ynh-deps depends on tcl; however:
2022-05-30 14:29:56,184: DEBUG -   Package tcl is not installed.
2022-05-30 14:29:56,184: DEBUG -  lxd-ynh-deps depends on ebtables; however:
2022-05-30 14:29:56,184: DEBUG -   Package ebtables is not installed.
2022-05-30 14:29:56,184: DEBUG -  lxd-ynh-deps depends on libapparmor-dev; however:
2022-05-30 14:29:56,185: DEBUG -   Package libapparmor-dev is not installed.
2022-05-30 14:29:56,185: DEBUG -  lxd-ynh-deps depends on libseccomp-dev; however:
2022-05-30 14:29:56,185: DEBUG -   Package libseccomp-dev is not installed.
2022-05-30 14:29:56,186: DEBUG -  lxd-ynh-deps depends on thin-provisioning-tools; however:
2022-05-30 14:29:56,186: DEBUG -   Package thin-provisioning-tools is not installed.
2022-05-30 14:29:56,186: DEBUG -  lxd-ynh-deps depends on btrfs-progs; however:
2022-05-30 14:29:56,186: DEBUG -   Package btrfs-progs is not installed.
2022-05-30 14:29:56,187: DEBUG -  lxd-ynh-deps depends on libattr1-dev; however:
2022-05-30 14:29:56,187: DEBUG -   Package libattr1-dev is not installed.
2022-05-30 14:29:56,187: DEBUG -  lxd-ynh-deps depends on libuv1; however:
2022-05-30 14:29:56,188: DEBUG -   Package libuv1 is not installed.
2022-05-30 14:29:56,188: DEBUG -  lxd-ynh-deps depends on lxc; however:
2022-05-30 14:29:56,188: DEBUG -   Package lxc is not installed.
2022-05-30 14:29:56,188: DEBUG -  lxd-ynh-deps depends on tcl8.6; however:
2022-05-30 14:29:56,189: DEBUG -   Package tcl8.6 is not installed.
2022-05-30 14:29:56,189: DEBUG - 
2022-05-30 14:29:56,189: DEBUG - Setting up lxd-ynh-deps (5.1~ynh1) ...
2022-05-30 14:29:56,790: DEBUG - + ynh_package_install --fix-broken
2022-05-30 14:29:56,790: DEBUG - + ynh_apt --no-remove --option Dpkg::Options::=--force-confdef --option Dpkg::Options::=--force-confold install --fix-broken
2022-05-30 14:29:56,791: DEBUG - + ynh_wait_dpkg_free
2022-05-30 14:29:56,985: DEBUG - + return 0
2022-05-30 14:29:56,985: DEBUG - + LC_ALL=C
2022-05-30 14:29:56,985: DEBUG - + DEBIAN_FRONTEND=noninteractive
2022-05-30 14:29:56,985: DEBUG - + apt-get --assume-yes --quiet -o=Acquire::Retries=3 -o=Dpkg::Use-Pty=0 --no-remove --option Dpkg::Options::=--force-confdef --option Dpkg::Options::=--force-confold install --fix-broken
2022-05-30 14:29:57,594: DEBUG - Reading package lists...
2022-05-30 14:29:57,869: DEBUG - Building dependency tree...
2022-05-30 14:29:57,869: DEBUG - Reading state information...
2022-05-30 14:29:58,021: DEBUG - Correcting dependencies... Done
2022-05-30 14:29:58,227: DEBUG - The following additional packages will be installed:
2022-05-30 14:29:58,227: DEBUG -   arch-test attr bridge-utils btrfs-progs busybox-static cloud-image-utils
2022-05-30 14:29:58,228: DEBUG -   debootstrap distro-info ebtables genisoimage libacl1-dev libapparmor-dev
2022-05-30 14:29:58,228: DEBUG -   libattr1-dev libcap-dev liblxc1 liblz4-dev libpam-cgfs libseccomp-dev
2022-05-30 14:29:58,229: DEBUG -   libsqlite3-dev libtcl8.6 libudev-dev libuv1 libuv1-dev lxc lxc-dev
2022-05-30 14:29:58,230: DEBUG -   lxc-templates lxcfs pkg-config qemu-utils squashfs-tools tcl tcl8.6
2022-05-30 14:29:58,230: DEBUG -   thin-provisioning-tools uidmap uuid-runtime
2022-05-30 14:29:58,233: DEBUG - Suggested packages:
2022-05-30 14:29:58,233: DEBUG -   duperemove cloud-utils-euca ubuntu-archive-keyring squid-deb-proxy-client
2022-05-30 14:29:58,233: DEBUG -   shunit2 wodim cdrkit-doc manpages-dev seccomp sqlite3-doc python3-lxc
2022-05-30 14:29:58,234: DEBUG -   qemu-block-extra tcl-tclreadline
2022-05-30 14:29:58,269: DEBUG - The following packages will be REMOVED:
2022-05-30 14:29:58,272: DEBUG -   busybox
2022-05-30 14:29:58,273: DEBUG - The following NEW packages will be installed:
2022-05-30 14:29:58,273: DEBUG -   arch-test attr bridge-utils btrfs-progs busybox-static cloud-image-utils
2022-05-30 14:29:58,273: DEBUG -   debootstrap distro-info ebtables genisoimage libacl1-dev libapparmor-dev
2022-05-30 14:29:58,275: DEBUG -   libattr1-dev libcap-dev liblxc1 liblz4-dev libpam-cgfs libseccomp-dev
2022-05-30 14:29:58,275: DEBUG -   libsqlite3-dev libtcl8.6 libudev-dev libuv1 libuv1-dev lxc lxc-dev
2022-05-30 14:29:58,276: DEBUG -   lxc-templates lxcfs pkg-config qemu-utils squashfs-tools tcl tcl8.6
2022-05-30 14:29:58,276: DEBUG -   thin-provisioning-tools uidmap uuid-runtime
2022-05-30 14:29:58,293: DEBUG - 0 upgraded, 35 newly installed, 1 to remove and 21 not upgraded.
2022-05-30 14:29:58,294: DEBUG - E: Packages need to be removed but remove is disabled.
2022-05-30 14:29:58,296: DEBUG - ++ cat /tmp/tmp.emC1brpNLq/dpkg_log
2022-05-30 14:29:58,297: DEBUG - ++ grep -oP '(?<=-ynh-deps depends on ).*(?=; however)'
2022-05-30 14:29:58,297: DEBUG - ++ tr '\n' ' '
2022-05-30 14:29:58,303: DEBUG - + local 'problematic_dependencies=attr libacl1-dev libcap-dev liblxc1 lxc-dev libsqlite3-dev libudev-dev liblz4-dev libuv1-dev pkg-config squashfs-tools tcl ebtables libapparmor-dev libseccomp-dev thin-provisioning-tools btrfs-progs libattr1-dev libuv1 lxc tcl8.6 '
2022-05-30 14:29:58,303: DEBUG - + [[ -n attr libacl1-dev libcap-dev liblxc1 lxc-dev libsqlite3-dev libudev-dev liblz4-dev libuv1-dev pkg-config squashfs-tools tcl ebtables libapparmor-dev libseccomp-dev thin-provisioning-tools btrfs-progs libattr1-dev libuv1 lxc tcl8.6  ]]
2022-05-30 14:29:58,304: DEBUG - + ynh_package_install attr libacl1-dev libcap-dev liblxc1 lxc-dev libsqlite3-dev libudev-dev liblz4-dev libuv1-dev pkg-config squashfs-tools tcl ebtables libapparmor-dev libseccomp-dev thin-provisioning-tools btrfs-progs libattr1-dev libuv1 lxc tcl8.6 --dry-run
2022-05-30 14:29:58,304: DEBUG - + sed --quiet '/Reading state info/,$p'
2022-05-30 14:29:58,305: DEBUG - + ynh_apt --no-remove --option Dpkg::Options::=--force-confdef --option Dpkg::Options::=--force-confold install attr libacl1-dev libcap-dev liblxc1 lxc-dev libsqlite3-dev libudev-dev liblz4-dev libuv1-dev pkg-config squashfs-tools tcl ebtables libapparmor-dev libseccomp-dev thin-provisioning-tools btrfs-progs libattr1-dev libuv1 lxc tcl8.6 --dry-run
2022-05-30 14:29:58,305: DEBUG - + ynh_wait_dpkg_free
2022-05-30 14:29:58,306: DEBUG - + grep -v 'fix-broken\|Reading state info'
2022-05-30 14:29:58,533: DEBUG - + return 0
2022-05-30 14:29:58,534: DEBUG - + LC_ALL=C
2022-05-30 14:29:58,534: DEBUG - + DEBIAN_FRONTEND=noninteractive
2022-05-30 14:29:58,534: DEBUG - + apt-get --assume-yes --quiet -o=Acquire::Retries=3 -o=Dpkg::Use-Pty=0 --no-remove --option Dpkg::Options::=--force-confdef --option Dpkg::Options::=--force-confold install attr libacl1-dev libcap-dev liblxc1 lxc-dev libsqlite3-dev libudev-dev liblz4-dev libuv1-dev pkg-config squashfs-tools tcl ebtables libapparmor-dev libseccomp-dev thin-provisioning-tools btrfs-progs libattr1-dev libuv1 lxc tcl8.6 --dry-run
2022-05-30 14:29:59,043: DEBUG - The following packages have unmet dependencies:
2022-05-30 14:29:59,043: DEBUG -  tcl8.6 : Depends: libtcl8.6 (>= 8.6.0) but it is not going to be installed
2022-05-30 14:29:59,044: DEBUG - + ynh_die '--message=Unable to install dependencies'
2022-05-30 14:29:59,073: DEBUG - Unable to install dependencies
2022-05-30 14:29:59,074: DEBUG - + ynh_exit_properly
2022-05-30 14:29:59,594: DEBUG - + ynh_clean_setup
2022-05-30 14:29:59,594: DEBUG - + ynh_exec_warn_less popd
2022-05-30 14:29:59,595: DEBUG - + [[ 1 -eq 1 ]]
2022-05-30 14:29:59,595: DEBUG - + [[ popd == *\ * ]]
2022-05-30 14:29:59,595: DEBUG - + popd
2022-05-30 14:29:59,596: DEBUG - /usr/share/yunohost/helpers.d/logging: Zeile 143: popd: Der Verzeichnisstapel ist leer.
2022-05-30 14:29:59,596: DEBUG - + ynh_secure_remove --file=
2022-05-30 14:29:59,629: DEBUG - + ynh_print_log 'ynh_secure_remove called with empty argument, ignoring.'
2022-05-30 14:29:59,630: DEBUG - + echo -e 'ynh_secure_remove called with empty argument, ignoring.'
2022-05-30 14:29:59,630: DEBUG - ynh_secure_remove called with empty argument, ignoring.
2022-05-30 14:29:59,631: DEBUG - + ynh_secure_remove --file=
2022-05-30 14:29:59,671: DEBUG - + ynh_print_log 'ynh_secure_remove called with empty argument, ignoring.'
2022-05-30 14:29:59,672: DEBUG - + echo -e 'ynh_secure_remove called with empty argument, ignoring.'
2022-05-30 14:29:59,672: DEBUG - ynh_secure_remove called with empty argument, ignoring.
2022-05-30 14:30:00,705: ERROR - Installation von lxd fehlgeschlagen: Im Installationsscript ist ein Fehler aufgetreten

**Comment by Tagadda** [comment]
This is not the good repository.
Please create a new issue at https://github.com/YunoHost-Apps/lxd_ynh/issues
